codecs/move-gst-plugins.sh

```diff
@@ -5,6 +5,11 @@ export GST_PLUGIN_SYSTEM_PATH=/app/lib/gstreamer-1.0
 export LD_LIBRARY_PATH=/app/lib/codecs/lib/:/app/lib/
 
 for i in /app/lib/gstreamer-1.0/*.so; do
+  if [ ${i##*/} == 'libgstfdkaac.so' ] ||
+     [ ${i##*/} == 'libgstva.so' ]; then
```

Comment:
Oops, this line is a no-op left over from when I was looking at `org.gnome.Totem`.